### ISSUE TITLE
Link coverage page from Records page

### ISSUE THREAD

**Issue by DeniseColbert** [FreeUKGen/FreeBMD2#527]
Coverage page (needs link in first paragraph removing): https://test3.freebmd.org.uk/cms/draft-coverage/draft-freebmd-progress

**Comment by richardofsussex** [comment]
I don't think I have access to this - is it a Refinery page? It certainly doesn't show up in my local test environment.

**Comment by AlOneill** [comment]
@richardofsussex 
The `/cms/` part of the address implies a Refinery page. 
BTW we have a protocol for updating Refinery pages that is designed to help ensure that updates are properly sequential and that they are made on the most up-to-date version of a page, etc. @DeniseColbert could you point Richard to the Google doc please — many thanks.

**Comment by AlOneill** [comment]
@DeniseColbert 
Looking a bit closer —
As the page concerned is a  DRAFT, and it is the _child_ of a DRAFT, then I doubt that our protocol covers the situation explicitly. However, looking at the pages concerned, they are both a long way from being ready for publication and I do not think that published versions of the pages exist. So … ?

**Comment by richardofsussex** [comment]
Sounds as though it should head to the documentation team, rather than to me.

**Comment by DeniseColbert** [comment]
Yes, Alison might think that you want access to Refinery to make changes to the pages but I think you just need to _see_ the page that's there. I thought anyone could see draft pages? Plus, you seem to be a Refinery user on Test 3 so I'm mightily confused as to why you can't see it!

**Comment by DeniseColbert** [comment]
Maybe it's your test environment causing the issue- can you try on a browser?

**Comment by richardofsussex** [comment]
I can see the page fine: I thought the issue was to remove a link that it contains.

**Comment by DeniseColbert** [comment]
Ah right, sorry for my misunderstanding. It's a placeholder link, and we're a way off having this ready yet. I think you were going to look at getting the coverage charts from FreeBMD1 over to FreeBMD2? My apologies for not noting that in the comments, if so.

**Comment by richardofsussex** [comment]
Yes, that's a separate issue (#517). As I say there, I think we need to agree the design and content of these pages before trying to produce them.

**Comment by KristinaGadzhieva** [comment]
To investigate what information can be showed on the Records main page

**Comment by KristinaGadzhieva** [comment]
@richardofsussex @nurifreembd How about including a question in user testing on whether users need coverage charts on FreeBMD?

**Comment by nurifreembd** [comment]
Yes you can to get their feedback

**Comment by KristinaGadzhieva** [comment]
For MVP implemented in #517.
